source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -74,6 +74,22 @@ void SwiftUserExpression::DidFinishExecuting() {
   }
 }
 
+static CompilerType getBoundOrUnboundType(CompilerType &valobj_type,
+                                          uint32_t idx) {
+  CompilerType key_type;
+  switch (valobj_type.GetTemplateArgumentKind(idx)) {
+  case lldb::eBoundGenericKindType:
+    key_type = valobj_type.GetBoundGenericType(idx);
```

Comment:
perhaps just `return valobj_type.GetBoundGenericType(idx);`?

Reply:
I definitely agree. I can put some more comments, and in general the whole support needs to be more commented. I'll refine this once somebody who understands the swift support in lldb better than I do signs off the general approach. Thanks!